core/README.md

```diff
@@ -156,3 +204,113 @@ the [overview](../overview.md#example-landing-page) document.
 ## Extensions
 
 None.
+
+## Structuring Catalog Hierarchies
+
+A STAC API is more useful when it presents a complete `Catalog` representation of all the data contained in the
+API, such that all `Item` objects can be reached by transitively traversing `child` and `item` link relations from
+the root. This property of being able to reach all Items in this way is formalized in the
+`STAC API - Browseable` conformance class, but any Catalog can be structured for hierarchical traversal. 
+Implementers who have search as their primary use case should consider also implementing this
+alternate view over the data by presenting it as a directed graph of catalogs, where the `child` link relations typically
+form a tree, and where each catalog can be retrieved with a single request (e.g., each Catalog JSON is small enough that
+it does not require pagination).
+
+While OAFeat requires that all Items must be part of a Collection, this does not mean that the Collection needs to be
+part of the browseable tree. If they are part of the tree, it is recommended that there only be one Collection in a
+path through the tree, and that sub-collections not be used.
```

Comment:
I'm not sure what sub-collections means here? May be worth fleshing out some.

Reply:
I removed that word here and one other place and explicitly used "collection with a child collection"